frontend/src/pages/PropertyDetails.jsx

```
@@ -429,6 +412,182 @@ const PropertyDetails = () => {
+            maxDate={maxDate}
+            excludeDateIntervals={excludeIntervals}
+            placeholderText={`Select ${label.toLowerCase()} date`}
+            className="w-full pl-3 pr-3 py-2 border rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500"
+          />
+        </div>
+    );
+      
+    const GuestInput = ({ value, onChange, max }) => (
+        <div>
+          <label className="block text-gray-700 mb-2">Guests</label>
+          <div className="relative">
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)